index.html

```diff
@@ -21,6 +21,10 @@ <h2 class="logo">Awesome Books</h2>
       </nav>
```

Comment:
- [ ]Nice job however the `li` is not switching between links in the `navbar`.

Reply:
Thank you, fixed it already
I did not know it was not working on deployed version